examples/ChatQnA/deploy/xeon.md

```diff
@@ -78,19 +72,33 @@ export https_proxy=${your_http_proxy}
 
 ## Prepare (Building / Pulling) Docker images
 
-This step will involve building/pulling ( maybe in future) relevant docker
+This step will involve building/pulling  relevant docker
 images with step-by-step process along with sanity check in the end. For
 ChatQnA, the following docker images will be needed: embedding, retriever,
 rerank, LLM and dataprep. Additionally, you will need to build docker images for
 ChatQnA megaservice, and UI (conversational React UI is optional). In total,
 there are 8 required and an optional docker images.
 
-The docker images needed to setup the example needs to be build local, however
-the images will be pushed to docker hub soon by Intel.
-
 ### Build/Pull Microservice images
 
-From within the `GenAIComps` folder
+::::{tab-set}
+
+:::{tab-item} Pull
+:sync: Pull
+
+Ifyou decide to pull the docker containers and not build them locally, 
+we can proceed to the next step where all the necessary containers will 

```

Comment:
It's not obvious what the "next step" is.  If it's the next heading (Build Dataprep Image), you should say that instead of "next step".  

Also the sentence starts with "If you..." and continues with "we can".  Change "we can" to "you can".